run_test.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
```

Comment:
can you create a new repo to host run_test.py, test.py, and test_attn.mlir and put the link in a comment in this PR? we will eventually use the C++ attention e2e test.

Reply:
See https://github.com/rohan-tan-bhowmik/iree-masked-attention-test